mod-di-converter-storage-server/src/main/java/org/folio/services/migration/ProfileMigrationServiceImpl.java

```diff
@@ -47,6 +49,14 @@ public Future<Boolean> migrateDataImportProfiles(Map<String, String> headers, Co
         if (isRowCount == 0) {
           return profileWrapperDao.checkIfDataInTableExists(tenantId)
             .compose(isDataPresent -> processMigration(isDataPresent, tenantId));
```

Comment:
can you change the name of the `processMigration` method to `processWrapperMigration`?

Reply:
Reading the PR further, `processMigration` is the right name since it does both wrapper and associations migrations.